### PR TITLE
Support PagerDuty Webhook

### PR DESCRIPTION
**Pull request by kirikiriyamama**
https://www.pagerduty.com/

_1__idobata

### CODE REVIEW

File: lib/hooks/pagerduty/hook.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Idobata::Hook
+  class Pagerduty < Base
+    screen_name 'PagerDuty'
+    icon_url    hook_image_url('icon.png')
```
Comment (ursm) [inline lib/hooks/pagerduty/hook.rb]: `icon320x320.jpeg`?

File: lib/hooks/pagerduty/templates/default.html.haml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- payload.messages.each do |message|
+  %p
+    %span{class: "label label-#{label_type(message.data.incident.status)}"}= message.data.incident.status
+    %b= "#{message.data.incident.service.name}:"
```
Comment (ursm) [inline lib/hooks/pagerduty/templates/default.html.haml]: Shorthand:

``` haml
%b #{message.data.incident.service.name}:
```

File: lib/hooks/pagerduty/hook.rb (hunk 1 above)
Comment (kirikiriyamama) [inline lib/hooks/pagerduty/hook.rb]: Oops! I have fixed it :bow:

File: lib/hooks/pagerduty/templates/default.html.haml (hunk 2 above)
Comment (kirikiriyamama) [inline lib/hooks/pagerduty/templates/default.html.haml]: It's cool! I have applied it.

**Comment by ursm** [comment]
Thank you very much! It seems to work very well.
We'd release it after this is tested in the staging environment just in case. Stay tuned.

**Comment by kirikiriyamama** [comment]
thx :sparkles:

**Comment by ursm** [comment]
:shipit:

**Pull request merged**